data/json/items/ammo/762.json

```diff
@@ -4,7 +4,7 @@
     "type": "AMMO",
     "ascii_picture": "762_m87",
     "description": "Imported 7.62x39mm M67-style ammunition with a 124gr jacketed, lead core bullet.  Thankfully this loading has non-corrosive primers, and also features a reloadable brass case.",
-    "name": { "str_sp": "7.62x39mm FMJ" },
+    "name": { "str_sp": "7.62x39mm Soviet FMJ" },
```

Comment:
Same here. Please don't put Soviet there, it feels clunky and doesn't add anything.